Frameworks/UIKit/StarboardXaml/StarboardXaml.cpp

```diff
@@ -223,4 +228,15 @@ void UIApplicationMainTest() {
     _ApplicationLaunch(ActivationTypeNone, nullptr);
 }
 
+UIKIT_EXPORT
+void UIApplicationActivationTest(IInspectable* activationArgs, wchar_t* delegateName){
+    // Initialize COM on this thread
+    ::CoInitializeEx(nullptr, COINIT_MULTITHREADED);
+
+    // Register tracelogging
+    TraceRegister();
+	g_delegateClassName = ref new Platform::String(delegateName);
```

Comment:
test

---
In reply to: [69815541](https://github.com/Microsoft/WinObjC/pull/603#discussion_r69815541) [](ancestors = 69815541)